src/components/ResourceList/components/BulkActions/BulkActions.tsx

```diff
@@ -204,6 +206,8 @@ class BulkActions extends React.PureComponent<CombinedProps, State> {
       polaris: {intl},
     } = this.props;
 
+    const actionSections = this.actionSections();
```

Comment:
This got called a whole load of times, so instead of computing the actionSections repeatedly, lets do it once